source/statements.tex

```diff
@@ -1031,6 +1042,48 @@
 \indextext{label}%
 label\iref{stmt.label} located in the current function.
 
+\rSec1[stmt.contract.assert]{Assertion statement}
+
+\begin{bnf}
+\nontermdef{assertion-statement}\br
+    \terminal{contract_assert} \opt{attribute-specifier-seq} \terminal{(} conditional-expression \terminal{)} \terminal{;}
+\end{bnf}
+
+\pnum
+\indexdefn{contract assertion!statement|see{assertion, statement}}
+\indextext{assertion!statement}
+An \grammarterm{assertion-statement}
+introduces a contract assertion\iref{basic.contract}.
+The optional \grammarterm{attribute-specifier-seq}
+appertains to the introduced contract assertion.
+
+\pnum
+The predicate\iref{basic.contract.general}
+of an \grammarterm{assertion-statement}
+is its \grammarterm{conditional-expression}
+contextually converted to \tcode{bool}.
+
+\pnum
+The evaluation of consecutive \grammarterm{assertion-statement}s
+is an evaluation in sequence\iref{basic.contract.eval} of
+the contract assertions introduced
+by those \grammarterm{assertion-statement}s.
+\begin{note}
+A sequence of \grammarterm{assertion-statement}s
+can thus be repeatedly evaluated as a group.  %JMB: Should we remove the "thus"?
```

Comment:
It's in the paper, it's in a note, and it's a consequence from the "evaluated in sequence" normative text.  Leave it in; remove JMB.

Reply:
done